Makefile

```diff
@@ -54,6 +60,20 @@ else
  GOBIN_SHARED_LIB_CGO_LDFLAGS := CGO_LDFLAGS="-Wl,-soname,libstatus.so.0"
 endif
 
+ifeq ($(OS),android)
+ MOBILE_BUILD_MODE := c-shared
+ MOBILE_LIB_EXT := so
+ MOBILE_BUILD_ENV := GOFLAGS="" \
+  CC="$(ANDROID_NDK_ROOT)/toolchains/llvm/prebuilt/$(HOST_OS)-x86_64/bin/$(ARCH)-linux-android$(ANDROID_API)-clang" \
+  AR="$(ANDROID_NDK_ROOT)/toolchains/llvm/prebuilt/$(HOST_OS)-x86_64/bin/llvm-ar" \
+  CGO_CFLAGS="-Os -flto" CGO_LDFLAGS="-Os -flto" CGO_ENABLED=1 GOOS="$(OS)" GOARCH="$(MOBILE_GOARCH)"
+else ifeq ($(OS),ios)
+ MOBILE_BUILD_MODE := c-archive
+ MOBILE_LIB_EXT := a
+ MOBILE_CC := $(shell xcrun --sdk $(IPHONE_SDK) --find clang)
+ MOBILE_BUILD_ENV := GOFLAGS="" CC="$(MOBILE_CC)" CGO_CFLAGS="-Os -flto -arch $(ARCH) -isysroot $(shell xcrun --sdk $(IPHONE_SDK) --show-sdk-path) -miphoneos-version-min=$(IOS_TARGET) -fembed-bitcode" CGO_LDFLAGS="-Os -flto" CGO_ENABLED=1 GOOS="$(OS)" GOARCH="$(MOBILE_GOARCH)"
+endif
```

Comment:
We can avoid most of this if we create separate `statusgo-android-library` and  `statusgo-ios-library` targets.